import_email_events.py

```diff
@@ -0,0 +1,24 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    flow_id VARCHAR(64) DISTKEY ENCODE lzo,
+    domain VARCHAR(40) ENCODE lzo,
+    template VARCHAR(64) ENCODE lzo,
+    type VARCHAR(64) NOT NULL ENCODE lzo,
+    complaint VARCHAR(64) ENCODE lzo,
+    bounced VARCHAR(64) ENCODE lzo,
+    locale VARCHAR(64) ENCODE lzo
+"""
+
+COLUMNS = "flow_id, domain, template, type, complaint, bounced, locale"
```

Comment:
I think `complaint` and `bounced` are still the wrong way round? Looks like the lua script processes them in that order:

https://github.com/mozilla-services/puppet-config/blob/master/shared/modules/fxa/files/hekad/lua_outputs/fxa_email_csv.lua#L40-L48